app/system/language/en/lang.php

```diff
@@ -6,6 +6,13 @@
     'copyright' => 'Thank you for using <a target="_blank" href="http://tastyigniter.com">TastyIgniter</a>',
     'version' => '<b>Version:</b> %s',
 
+    'date_format_short' => 'd m Y',
+    'date_format_long' => 'l, jS F Y',
+    'time_format_short' => 'H:i',
+    'time_format_long' => 'h:i a',
+    'date_time_format_short' => 'd m Y H:i',
+    'date_time_format_long' => 'l, jS F Y \a\t h:i a',
+
```

Comment:
What do you think?

```
    'date_format' => 'd m Y',
    'date_format_long' => 'l, jS F Y',
    'time_format' => 'H:i',
    'time_format_long' => 'h:i a',
    'date_time_format' => 'd m Y H:i',
    'date_time_format_long' => 'l, jS F Y \a\t h:i a',
```

Also bear in mind that components use the moment.js date format style ie. DDDD, MMM YYYY, so we will need more lang strings for components